suite_test.go

```diff
@@ -279,17 +280,18 @@ func (suite *serviceBusSuite) publishAndReceiveMessage(testConfig publishReceive
 	suite.NoError(err)
 }
 
-func checkResultHandler(publishedMsg string, publishedMsgType string, ch chan<- bool) Handle {
-	return func(ctx context.Context, message string, messageType string) error {
-		if publishedMsg != message {
-			ch <- false
-			return errors.New("published message and received message are different")
-		}
-		if publishedMsgType != messageType {
-			ch <- false
-			return errors.New("published message type and received message type are different")
-		}
-		ch <- true
-		return nil
-	}
+func checkResultHandler(publishedMsg string, publishedMsgType string, ch chan<- bool) handle.Handler {
+	return handle.Handle(
+		func(ctx context.Context, message string, messageType string) handle.Handler {
```

Comment:
another handler using Error and Complete

Reply:
Should we get retry later in here or a different suite? Not that familiar.

Reply:
Yes, working on it. The checkbox isn't ticked yet in the description :)